src/main/java/io/github/caitlinrichard/tictactoe/Grid.java

```diff
@@ -0,0 +1,66 @@
+package io.github.caitlinrichard.tictactoe;
+
+public class Grid {
+	private int[] gridState = new int[9]; //holds values for grid cells, 0 = x, 1 = y, -1 = blank
+	
+	//default constructor, initiates blank grid
+	public Grid() {
+		for (int i = 0; i < gridState.length; i++) {
+			gridState[i] = -1;
+		}
+	}
+	
+	//returns boolean if the cell is already occupied with an X or O
+	public boolean isCellOccupied(int cell) {
+		boolean isCellOccupied;
+		if (gridState[cell] != -1) {
```

Comment:
```suggestion
		if (gridState[cell] != -1) {
```
I think this should technically be "if not 0 or 1". As it stands, gridState could be, say, 7 and it would be considered "occupied". But only 0 and 1 are occupied states. Not a huge deal one way or the other.

Reply:
made changes. return (cellState == 0 || cellState == 1);